### PR TITLE
Mini-showcase extensions in front page

### PR DESCRIPTION
**Pull request by humitos**
It uses card to add an animation/screenshot relevant to the extension itself trying to highlight what it looks like or what's about.

Using a 300x image sometimes is not enough, so there are some images that are not super great. We can iterate on them to change these later if required.

Screenshot_2022-12-01_12-56-36

📚 Live example: https://sphinx-extensions--35.org.readthedocs.build/en/35/

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -62,3 +142,17 @@ Visit https://sphinx-themes.org/ for a full list of live themes!
    :hidden:
 
    *
+
+.. seealso::
```
Comment (ericholscher) [inline docs/index.rst:146]: This is a great section 💯

File: docs/conf.py [hunk 2]
```diff
@@ -41,8 +41,18 @@
...
     'sphinx_last_updated_by_git',
+    'sphinx_design',
+    'sphinx-favicon',
 ]
 
+
+favicons = [
+   {
+      "rel": "icon",
+      "static-file": "favicon.svg",
```
Comment (ericholscher) [inline docs/conf.py:52]: Did you make the favicon? Very fancy :)

**Review by ericholscher — APPROVED** [review]
This is a great improvement. I do find the screenshots a little visually hard to parse, but there's already borders & shadow around them, so not sure much can be done there immediately. I do think integrating these screenshots onto the "detail" pages for the extensions could be a good next step, but having examples as we do for most of them is also great 💯

File: docs/conf.py (hunk 2 above)
Comment (humitos) [reply docs/conf.py:52]: Ha ha, yes. "Graphic Design is my passion" 😝

**Comment by humitos** [comment]
> I do find the screenshots a little visually hard to parse, but there's already borders & shadow around them, so not sure much can be done there immediately.

Agreed. The ones that have an arrow work a lot better in my opinion, because it directs my eyes exactly where I need.

I tried different approaches here: trying to have them look pretty similar, use the exact same dimensions, do not use other colors, etc --but it required a lot more work and the result was not way better. So, I decided to continue with this idea and see how it grows from here.

It works fine for now --definitely better than just text. I think we can find a way to make it work better even, but probably it requires more thinking and graphic skills 😄 

I need to still play with the ordering based on colors, importance and maybe highlight some of them in a different way. This will require some exploration and testing. Baby steps 👶🏼 

> I do think integrating these screenshots onto the "detail" pages for the extensions could be a good next step, but having examples as we do for most of them is also great 100

Yes. The detail pages need more work. I have some ideas in mind for this. However, I think having a _live example_ of the extension itself will work better than a screenshot/animation.
